frontend/src/components/Button/Button.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { TouchableOpacity, Text } from "react-native";
 import { styles } from "./ButtonStyles";
 
-export default Button = ({ onPress, title }) => {
+const Button = ({ onPress, title }) => {
```

Comment:
Interested in this change too. I am not aware of any difference. Is a linter or consistency issue?

Reply:
linter issue but also probably helps with consistency